src/packages/core/protocol/__tests__/account.spec.ts

```diff
@@ -31,14 +31,17 @@ describe('create new accounts', () => {
 				// 	locked: 0,
 				// 	usable: 0
 				// });
+				// Using a pre-defined address..
 
-				// Using a pre-defined address.. 
+				const resp = {total: 9999999999999, locked: 0, usable: 9999999999999};
 
-				let address = 'TTatyig2SCtmUEsgguKvxQQ421e6NULS';
-				let balance = await Account.getBalance(address, {host: 'https://explorer.nuls.services', base: ''});
-				let total = balance.total;
-				let locked = balance.locked;
-				let usable = balance.usable;
+				Account.getBalance = jest.fn((a, b) => {return Promise.resolve(resp)});
```

Comment:
What you're doing here is overwriting the `getBalance` function with a mock and then testing the mock function rather than the `getBalance`  function. You want to mock the `get` api function to fake the response from the api server.

Reply:
You're right mate. I had the issue that if I tried to mock the whole axios module, the tests would have failed, but luckily I found out I can only mock/spyOn a function. Fixed in the last commit.